specs/SensitivityLabel.md

```diff
@@ -0,0 +1,541 @@
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2* sender,
+      [in] ICoreWebView2SensitivityLabelEventArgs* args);
+}
+
+/// Extension of the ICoreWebView2 interface that provides sensitivity label
+/// change notification capabilities. This interface enables applications to
+/// monitor and respond to changes in the sensitivity classification of web
+/// content loaded in the WebView2 control. When sensitivity labels are
+/// detected, updated, or removed from web pages, the SensitivityLabelChanged
+/// event is raised.
+[uuid(ac4543d5-f466-5622-8b3b-24d3b195525c), object, pointer_default(unique)]
+interface ICoreWebView2_32 : IUnknown {
+  /// Adds an event handler for the `SensitivityLabelChanged` event.
+  /// Event raised when the sensitivity label classification of web page 
+  /// changes. web pages may report sensitivity labels via
+  /// [`Page Interaction Restriction Manager`](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PageInteractionRestrictionManager/explainer.md).
+  /// This event is triggered when the WebView2 control detects a change in 
+  /// the sensitivity labels associated with the currently loaded web page.
+  /// Changes can occur when navigating to a new page in the main frame,
+  /// when the existing page updates its sensitivity label information.
+  /// On navigation to a new page `SensitivityLabelChanged` event is raised
+  /// after the `NavigationCompleted` event. Applications can subscribe to
+  /// this event to receive notifications about sensitivity changes. The
+  /// event args provide the current label state and the complete collection
+  /// of detected sensitivity labels. the complete collection of detected
+  /// sensitivity labels.
+  HRESULT add_SensitivityLabelChanged(
+      [in] ICoreWebView2SensitivityLabelChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with 
+  /// `add_SensitivityLabelChanged`.
+  HRESULT remove_SensitivityLabelChanged(
+      [in] EventRegistrationToken token);
+
+}
+```
+### .NET/WinRT
+```c#
+enum CoreWebView2SensitivityLabelKind
+{
+    Unknown = 0,
+    Mip = 1,
+};
+
+enum CoreWebView2SensitivityLabelState
+{
+    None = 0,
+    Undetermined = 1,
+    Determined = 2,
+};
+
+runtimeclass CoreWebView2SensitivityLabelEventArgs
+{
+    // ICoreWebView2SensitivityLabelEventArgs members
+    IVectorView<CoreWebView2SensitivityLabel>  SensitivityLabels { get; };
+
+    CoreWebView2SensitivityLabelState SensitivityState { get; };
+}
+
+runtimeclass CoreWebView2SensitivityLabelCollectionView
+{
+    // ICoreWebView2SensitivityLabelCollectionView members
+    UInt32 Count { get; };
+
+    CoreWebView2SensitivityLabel GetValueAtIndex(UInt32 index);
+}
+
+runtimeclass CoreWebView2SensitivityLabel
+{
+    // ICoreWebView2SensitivityLabel members
+    CoreWebView2SensitivityLabelType LabelType { get; };
```

Comment:
It might be better to use "Kind" instead of "Type"
e.g. CoreWebView2SensitivityLabelKind and LabelKind.